namer/namer.cc

```diff
@@ -347,6 +347,7 @@ class SymbolFinder {
     // `private` with no arguments toggles the visibility of all methods below in the class def.
     // This tracks those as they appear.
     vector<optional<Modifier>> methodVisiStack = {nullopt};
+    bool inBlock = false;
```

Comment:
Look at the comment on line 339 in this file:

> Does not report any errors, which lets us cache its output.

You cannot add errors in this phase of Sorbet.

I would encourage you to look into doing this in rewriter/Flatten.cc or a new, dedicated `rewriter/` pass.

Reply:
good catch thank you!